sklearn/ensemble/tests/test_bagging.py

```diff
@@ -543,6 +543,16 @@ def test_base_estimator():
                                 random_state=0).fit(X_train, y_train)
     assert_true(isinstance(ensemble.base_estimator_, SVR))
 
+    # IsolationForest
+    X_train, X_test, y_train, y_test = train_test_split(boston.data,
+                                                        boston.target,
+                                                        random_state=rng)
+
+    ensemble = IsolationForest(n_jobs=3,
```

Comment:
I would move all the IsolationForest code to ensemble/iforest.py and put all the related tests in ensemble/tests/test_iforest.py